### DISCUSSION TITLE
User permissions

### DISCUSSION THREAD

**Discussion by SamSyntax** [snipe/snipe-it#12201] [Q&A]
Hi, we are currently trying to find an asset managment system for a project we run in our company. The key feature we are looking for is assigning permission to user/groups that allows to view assets which belong to particular company. As far as I went thgrough all the demo features I could not really find any way to implement this rule, however there is a tiny hint in the "Companies" tab informing us about some extended possibilites for granting permissions once the full support is purchased. My main question is - Would it be or is possible to achieve thing I mentioned earlier in the fully supported, purchased version of the program?  I provide a screenshot of mentioned information. 

![image](https://user-images.githubusercontent.com/95968905/205245209-8b3553dd-68b1-4ebd-a114-d1f45e09115d.png)

**Comment by brianhoganm** [comment]
SnipeIT is open source and free to install on-premise. No need to purchase anything. However, SnipeIT offers affordable hosting packages where they take care of the server set-up, installation, maintenance and upgrades if you go this route. 
To answer your question, you can enable **Full Multiple Companies Support** within SnipeIT options to allow certain access to the program for your users or groups. You should look at the demo again and click on a user to view the type of permissions you can set. I use SnipeIT to many several thousand assets and have many groups with special permissions that have either view only, edit, or delete, etc. Hope this helps!
